src/sql/workbench/services/accountManagement/browser/accountManagementService.ts

```diff
@@ -232,10 +236,11 @@ export class AccountManagementService implements IAccountManagementService {
 	}
 
 	/**
-	 * Retrieves all the accounts registered with ADS.
+	 * Retrieves all the accounts registered with ADS based on auth library in use.
 	 */
 	public getAccounts(): Promise<azdata.Account[]> {
-		return this._accountStore.getAllAccounts();
+		return this._accountStore.getAllAccounts()
```

Comment:
Did you go through and make sure nowhere else is using getAllAccounts incorrectly? May be worth looking into making it have a required param to specify the account lib to get accounts for

Reply:
Yes, all other paths are covered.